remote-query/remote-query-client/src/main/java/org/infinispan/query/remote/client/impl/MarshallerRegistration.java

```diff
@@ -41,7 +41,7 @@ public static void init(SerializationContext ctx) throws IOException {
    public static void registerProtoFiles(SerializationContext ctx) throws IOException {
       FileDescriptorSource fileDescriptorSource = new FileDescriptorSource();
       fileDescriptorSource.addProtoFile(QUERY_PROTO_RES, MarshallerRegistration.class.getResourceAsStream(QUERY_PROTO_RES));
-      fileDescriptorSource.addProtoFile(MESSAGE_PROTO_RES, MarshallerRegistration.class.getResourceAsStream(MESSAGE_PROTO_RES));
+      fileDescriptorSource.addProtoFile(WrappedMessage.PROTO_FILE, MarshallerRegistration.class.getResourceAsStream(WrappedMessage.PROTO_FILE));
```

Comment:
`WrappedMessage.PROTO_FILE` needs to be prepended with a "/" otherwise the following exception is thrown:

```java
Caused by: java.lang.IllegalArgumentException: contents cannot be null
	at org.infinispan.protostream.FileDescriptorSource.addProtoFile(FileDescriptorSource.java:101)
	at org.infinispan.query.remote.client.impl.MarshallerRegistration.registerProtoFiles(MarshallerRegistration.java:45)
	at org.infinispan.query.remote.client.impl.MarshallerRegistration.init(MarshallerRegistration.java:31)
	at org.infinispan.query.remote.impl.ProtobufMetadataManagerImpl.<init>(ProtobufMetadataManagerImpl.java:56)
	at org.infinispan.query.remote.impl.LifecycleManager.initProtobufMetadataManager(LifecycleManager.java:109)
	at org.infinispan.query.remote.impl.LifecycleManager.cacheManagerStarting(LifecycleManager.java:101)
	at org.infinispan.factories.GlobalComponentRegistry.preStart(GlobalComponentRegistry.java:267)
	at org.infinispan.factories.AbstractComponentRegistry.start(AbstractComponentRegistry.java:312)
	at org.infinispan.manager.DefaultCacheManager.internalStart(DefaultCacheManager.java:734)
	... 27 more

```

Reply:
Oops. The intention was to remove getResourceAsStream and use the alternative FileDescriptorSource.fromResources which treats all paths as absolute (and internally adds a leading / if needed).

Reply:
Updated.